cmd/common.go

```diff
@@ -237,6 +238,64 @@ func loadConfig(nodetype check.NodeType) string {
 	return filepath.Join(path, file)
 }
 
+func mapToCISVersion(kubeToCISMap map[string]string, kv string) string {
```

Comment:
```suggestion
func mapToBenchmark(kubeToBenchmark map[string]string, kv string) string {
```

Avoiding saying "CIS" to allow for things like Red Hat's document